.changeset/long-grapes-call.md

```diff
@@ -0,0 +1,6 @@
+---
+"@hashicorp/ember-flight-icons": major
+"@hashicorp/design-system-components": major
```

Comment:
Question: since in this PR we're just updating README files, in what technically this is a _major_ breaking change? 🤔

(just to understand)

Reply:
When cherry-picking from #1704 I failed to port the `package.json` changes – have updated the PR now.

Reply:
On the point of what constitutes the breaking change: it comes from us no longer guaranteeing that our packages work with Node 14. In practice this means our consumers may get a warning if they run their application with Node 14.

Reply:
> may get a warning if they run their application with Node 14.

FWIW I believe their apps will actually break in this case. The warning is for cases where you were using a _new_ version of Node.

Reply:
Ah, interesting. I got warnings (in other applications) where I was told that I'm using a deprecated Node version but it went through with the deployment. In any case, that's why we flag it as breaking 😁